projects/igniteui-angular/schematics/ng-add/index.spec.ts

```diff
@@ -105,13 +106,15 @@ describe('ng-add schematics', () => {
     expect(testTs).not.toContain('import \'hammerjs\';');
   });
 
-  it('should add hammer.js in angular.json build options under scripts', async () => {
+  // Hammer is optional now.
+  xit('should add hammer.js in angular.json build options under scripts', async () => {
     await runner.runSchematic('ng-add', { normalizeCss: false }, tree);
     const ngJsonConfigResult = JSON.parse(tree.read('/angular.json').toString());
     expect(ngJsonConfigResult.projects.testProj.architect.build.options.scripts).toContain('./node_modules/hammerjs/hammer.min.js');
   });
 
-  it('should add hammer.js in angular.json test options under scripts', async () => {
+  // Hammer is optional now.
+  xit('should add hammer.js in angular.json test options under scripts', async () => {
```

Comment:
We _should_ drop actual the functionality that adds hammer to the angular.json scripts collections. Commenting out the tests doesn't disable the actual logic and I see no changes in the ng-add schematic

Otherwise, this test should stay active